src/snowflake/snowpark/modin/pandas/groupby.py

```diff
@@ -1244,6 +1277,11 @@ def apply(self, func, *args, **kwargs):
             return dataframe_result.squeeze(axis=1).rename(self._df.columns[-1])
         return dataframe_result
 
+    def get_group(self, name, obj=None):
```

Comment:
Have you tried checking if this solution works out of the box for SeriesGroupBy as well?

Reply:
It does not due to [SNOW-1478924](https://snowflakecomputing.atlassian.net/browse/SNOW-1478924).